lms/djangoapps/django_comment_client/management/commands/tests/test_rename_user.py

```diff
@@ -0,0 +1,128 @@
+        cls.mongo_comment_ids = []
+        cls.sql_user = UserFactory.create()
+        cls.rename_cmd = rename_user.Command()
+        cls.seed_mongo_comment()
+        cls.seed_mongo_user()
+
+    @classmethod
+    def seed_mongo_comment(cls):
+        '''
+        Seeds 1 test mongo comment object for each of the test mongo users
+        '''
+        comment_1 = {
+            "_type": "CommentThread",
+            "abuse_flaggers": [],
+            "anonymous": False,
+            "anonymous_to_peers": False,
+            "at_position_list": [],
+            "author_id": "1",
+            "author_username": cls.sql_user.username,
+            "body": "Architecto suscipit dolores. Velit enim quasi doloribus suscipit maxime laboriosam. Voluptatibus totam dolor dolorem dolorum placeat. Provident necessitatibus fugiat quia aut numquam repudiandae. Cum excepturi doloremque mollitia itaque eum praesentium.\n\nEx tempore maiores eum ea sed alias sint. Maiores veritatis commodi dolores aut et dolorem mollitia. Qui neque sed ea placeat.\n\nSit voluptas dicta et ut. Voluptas velit beatae veniam magni dolores. Est sed impedit numquam dolorum perspiciatis veniam. Temporibus inventore distinctio ratione labore nobis alias dolores. Est velit nihil dolorem qui nesciunt asperiores veritatis.",
+            "closed": False,
+            "comment_count": 9,
+            "commentable_id": "video_1",
+            "course_id": "MITx/6.002x/2012_Fall",
+            "created_at": datetime.datetime.strptime('22092013', "%d%m%Y"),
+            "historical_abuse_flaggers": [],
+            "last_activity_at": datetime.datetime.strptime('22092013', "%d%m%Y"),
+            "tags_array": [
+                "c#",
+                "2012",
+                "java-sucks"
+            ],
+            "thread_type": "discussion",
+            "title": "Dolore et molestias quia maxime eaque quos voluptatem et.",
+            "updated_at": datetime.datetime.strptime('22092013', "%d%m%Y"),
+            "visible": True,
+            "votes": {
+                "count": 7,
+                "down": [
+                    "1",
+                    "3",
+                    "4",
+                    "5",
+                    "7"
+                ],
+                "down_count": 5,
+                "point": -3,
+                "up": [
+                    "2",
+                    "6"
+                ],
+                "up_count": 2
+            }
+
+        }
+        cls.mongo_comment_ids.append(cls.db.contents.insert(comment_1))
+
+    @classmethod
+    def seed_mongo_user(cls):
+        '''
+        Seeds 3 mongo test users for the tests that follow
+        '''
+        user_1 = {
+            "default_sort_key": "date",
+            "notification_ids": [],
+            "external_id": "test_user_1",
+            "username": cls.sql_user.username
+        }
+        cls.mongo_user_ids.append(cls.db.users.insert(user_1))
+
+    def test_data_seed(self):
```

Comment:
since an OperationFailure will be raised, as per the documentation, I will go ahead and remove this test, as we can be sure, as you said, that our data is sane as long as Pymongo did not throw an error